eva/optimizer/rules/parser_visitor/_rename_statement.py

```diff
@@ -0,0 +1,54 @@
+# coding=utf-8
+# Copyright 2018-2020 EVA
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from eva.parser.rename_statement import RenameTableStatement
+from eva.parser.table_ref import TableRef, TableInfo
+
+from eva.parser.evaql.evaql_parserVisitor import evaql_parserVisitor
+from eva.parser.evaql.evaql_parser import evaql_parser
+from eva.utils.logging_manager import logger
+
+# Modified
+##################################################################
+# RENAME STATEMENTS
+##################################################################
+
+
+class RenameTable(evaql_parserVisitor):
+    def visitOldtableName(self, ctx: evaql_parser.OldtableNameContext):
```

Comment:
We don't need new constructs for newTable and oldTable. Rather we can just use the tableName for both

Reply:
Delete this file